Back to attributes link on attributes edit page

AnwerAR opened the pull request:
see #15059 The issue was about editing product attributes. Added "Back to attributes" link.

Commit:
see #15059 The issue was about editing product attributes. Added back to attributes link.

mikejolley commented on includes/admin/class-wc-admin-attributes.php:
```diff
@@ -202,7 +202,7 @@ private static function process_edit_attribute() {
 		}
 
-		echo '<div class="updated"><p>' . __( 'Attribute updated successfully', 'woocommerce' ) . '</p></div>';
+		echo '<div class="updated"><p>' . __( 'Attribute updated successfully', 'woocommerce' ) . '</p><p><a href="'. admin_url( 'edit.php?post_type=product&amp;page=product_attributes' ).'">&larr; ' . __( 'Back to Attributes', 'woocommerce' ) . '</a></p></div>';
```
Wrap URL in `esc_url`

mikejolley replied:
Also needs space here `).'"`

AnwerAR replied:
Should i replace `admin_url` with `esc_url` or can wrap `admin_url` into `esc_url`?

AnwerAR committed:
Updates admin_url to esc_url

mikejolley commented:
Not sure what you're doing here... ecs_url is not valid, and I mean't WRAP the admin_url with an escaping function.

AnwerAR committed:
Added admin_url again wrapping inside esc_url

The function works without admin_url in my previous commit. but i have added admin_url again to make sure it points to correct location.

mikejolley commented:
Same typo. `ecs_url`?

claudiosanches commented:
We need to avoid using arrows `&larr; `, this is bad for RTL languages, even worse that is not able not even to translate it different.

AnwerAR commented:
@mikejolley OMG, sorry for typo. it's correct on my localhost. but i do not push changes from localhost. wanna understand `Grunt` tasks first. 
@claudiulodro shall i completely remove `&larr;` or need to be wrapped in `gettext` function along with message? what about `is_rtl() ? '&larr;' : '&rarr;';`?

AnwerAR committed:
Correctted typo of esc function and removed icon

correction of spell mistake and removal of &larr due to translation and RTL concerns. though WP and WC itself using &larr on some places.
